sycl-fusion/jit-compiler/lib/fusion/FusionHelper.cpp

```diff
@@ -92,7 +92,7 @@ Expected<std::unique_ptr<Module>> helper::FusionHelper::addFusedKernel(
     {
       const auto MDFromND = [&LLVMCtx](const auto &ND) {
         auto MDFromIndices = [&LLVMCtx](const auto &Ind) -> Metadata * {
-          std::array<Metadata *, jit_compiler::Indices{}.size()> MD{nullptr};
+          std::array<Metadata *, 3> MD{nullptr};
```

Comment:
Should we add a static `constexpr` function to `Indices`, so we do not have to use the magic number `3`, but instead can use `Indices::size()`?

Reply:
```suggestion
          std::array<Metadata *, jit_compiler::Indices::size()> MD{nullptr};
```